.dockerignore

```diff
@@ -1,13 +1,11 @@
-
-/_build
-/db
-/deps
+_build
```

Comment:
This is required to make things work: https://github.com/docker/compose/issues/1607